routes/store.php

```diff
@@ -166,6 +166,19 @@ function () {
                 'as' => 'store.vouchers.mvl.export',
                 'uses' => 'VoucherController@exportMasterVoucherLog',
             ]);
+
+            // Intern 4 test route
+            Route::get('/vouchers/historical', [
+                'as' => 'store.vouchers.mvl.historical',
+                'uses' => 'VoucherController@listVoucherLogs',
+            ]);
+
+            // Intern 4 test route
```

Comment:
Probably clean up the comments here, replace them with something that explains what they do.